quodlibet/config.py

```diff
@@ -189,6 +189,8 @@
         # (0 = disabled i.e. arithmetic mean)
         "bayesian_rating_factor": "0.0",
 
+        "weight_smoothing_factor": "10",
```

Comment:
Some comments like the above might be helpful - in particular what value turns it off

Reply:
Good point!